langchain/prompts/base.py

```diff
@@ -144,21 +149,29 @@ def validate_variable_names(cls, values: Dict) -> Dict:
             )
         return values
 
-    def partial(self, **kwargs: Union[str, Callable[[], str]]) -> BasePromptTemplate:
```

Comment:
This could use documentation in the method itself -- I'm not sure that everyone know what `partial` does with a regular str variable. Using a partial template is more complex conceptually